edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -159,10 +159,10 @@
         <form id="enter-grade-form">
           <input id="module_id-input" type="hidden" name="module_id"/>
           <input id="submission_id-input" type="hidden" name="submission_id"/>
-          <div>Grade for <span id="student-name"/></div>
-          <div>Grade: <input id="grade-input" name="grade"/>
+          <div>{% trans "Grade for" %}<span id="student-name"/></div>
```

Comment:
Please see http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html particularly the section on "General Internationalization Rules" and "Style Guidelines"